ci/index.html

```diff
@@ -1,9 +1,41 @@
 <html>
-  <head>
-  </head>
-  <body>
-    <h1>CI Build Job Results</h1>
-    <p><a href='artifacts/index.html'>artifacts</a></p>
-    <p><a href='job.log'>build log</a></p>
-  </body>
+<head>
+    <style type="text/css">
+    body {
+        font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
+    }
+    </style>
+</head>
+<body>
+<h1>CI Build Job Results</h1>
+<p><a href='job.log'>build log</a></p>
+
+<h1>Hail</h1>
+<object type="text/html" data="artifacts/hail.html"></object>
+<hr>
+
+<h1>Batch</h1>
+<object type="text/html" data="artifacts/batch.html"></object>
+<hr>
+
+<h1>Batch</h1>
```

Comment:
h1s are wrong for this one and ones after

Reply:
thanks